<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Следующая</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Разница</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="line_main_date">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая дата</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set_date">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить дату</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_prev_date">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_is_leap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Високосный</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_next_date">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_leap_date">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_prev_day">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущий день</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_week_date">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_week_num">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер недели</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_next_day">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Следующий день</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>751</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Создать/редактировать дату</string>
    </property>
    <widget class="QPushButton" name="on_buttonCreateSave_clicked">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="on_buttonCreateAdd_clicked">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_create_n">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>N</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_create_n">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_day">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>День</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_month">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Месяц</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_year">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Год</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_create_day">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_create_month">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_create_year">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Save_to_File">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>30</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить в файл</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SaveFileMessage">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>341</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть файл</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
